app/Api/Controllers/WebApiController.php

```diff
@@ -34,4 +36,25 @@ public function users(Request $request): JsonResponse
             'data' => $request->input(),
         ], 501);
     }
+
+    public function login(Request $request): JsonResponse
```

Comment:
Is it possible to add unit tests for this? I would really like everything in the V2 API to have good test coverage.

Reply:
Yes - certainly. Pending if we decide to keep this in here, I'm happy to add code coverage.